homeassistant/components/growatt_server/__init__.py

```diff
@@ -1 +1,57 @@
 """The Growatt server PV inverter sensor integration."""
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.helpers.typing import HomeAssistantType
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.config_validation import PLATFORM_SCHEMA
+from homeassistant.const import CONF_NAME, CONF_USERNAME, CONF_PASSWORD
+from .const import DOMAIN, CONF_PLANT_ID, DEFAULT_PLANT_ID, DEFAULT_NAME
+
+
+CONFIG_SCHEMA = vol.Schema(
```

Comment:
I cannot seem to get the config schema working as i want.
Right now i'm getting 
```
Invalid config for [growatt_server]: [username] is an invalid option for [growatt_server]. Check: growatt_server->username. (See ?, line ?). Please check the docs at https://home-assistant.io/integrations/growatt_server/
```
The config i'm using is:
```
 growatt_server:
   - username: growatt_user
     password: growatt_pass
```

Reply:
I'm not sure what's wrong. It looks good to me.